### PR TITLE
Feat: Added Playlist Page

### PR DESCRIPTION
**Pull request by KanekarSnehal**
### Overview
---
Added playlist page where user can view all available playlist
user can also add new playlist
user can select a particular playlist to see the videos present in it
added playlist modal so that user can add new playlist and also add a particular video to a any number of playlist

### Preview
---
https://deploy-preview-6--magnet-library.netlify.app/

### CODE REVIEW

**Review by sharathnair9999 — APPROVED** [review]
Looks clean. 

Suggestion : You can add alert or toast messages for better user understanding

File: src/services/playlist.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  GET_PLAYLISTS,
+  ADD_PLAYLIST,
+  DELETE_PLAYLIST,
+  GET_PLAYLIST_BY_ID,
+  ADD_VIDEO_TO_PLAYLIST,
+  DELETE_VIDEO_FROM_PLAYLIST,
+} = playlistConstants;
+
+const config = {
+  headers: { authorization: localStorage.getItem("token") },
```
Comment (deepak29-git) [inline src/services/playlist.js:15]: suggestion: you can make a function `getToken` and use it everywhere

**Review by deepak29-git — APPROVED** [review]

**Comment by KanekarSnehal** [comment]
> Looks clean.
> 
> Suggestion : You can add alert or toast messages for better user understanding

Thank you for the suggestion, will add it in upcoming PR's

File: src/services/playlist.js (hunk 1 above)
Comment (KanekarSnehal) [reply src/services/playlist.js:15]: Function won't be efficient I guess, I would have to call the function inside api calls, that's why used object, but I have to make the object constant and use it everywhere. Thanks for pointing out.

**Pull request merged**